Infrastructure/ApplicationDbContext.cs

```diff
@@ -0,0 +1,43 @@
+﻿using LearnNet_CatalogService.Data.Entities;
+using Microsoft.EntityFrameworkCore;
+using System.Reflection.Metadata;
+
+namespace LearnNet_CatalogService.DataAccessSQL
+{
+    public class ApplicationDbContext : DbContext
+    {
+        public DbSet<Category> Categories { get; set; }
+        public DbSet<Product> Products { get; set; }
+
+        public ApplicationDbContext(DbContextOptions<ApplicationDbContext> options)
+            : base(options)
+        {
+
+        }
+
+        protected override void OnModelCreating(ModelBuilder modelBuilder)
+        {
+            modelBuilder.Entity<Category>()
+                .Property(b => b.Name)
+                    .IsRequired()
+                    .HasMaxLength(50);
+
+            modelBuilder.Entity<Product>()
```

Comment:
these calls can be chained

Reply:
I don't think so, model builder doesn't support multiple properties in one call as I see